### PR TITLE
Using Docker 

### PR DESCRIPTION
**Pull request by greenmind-sec**
Add new pull request 
- Using Docker
- Modify README

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+RUN apt-get update -y
+
+RUN apt-get install python3-pip -y
+
+RUN apt-get install php7.0 -y
+
+RUN apt-get install unzip -y
+
+WORKDIR /root
```
Comment (A1S0N) [inline Dockerfile]: docker support

**Comment by A1S0N** [comment]
add docker support

**Comment by aljokarmohamed** [comment]
Hello my friends ... I need help ... Please .. I need to penetrate Emil but I did not appreciate .... Can you help me

**Comment by aljokarmohamed** [comment]
Hello my friends ... I need help ... Please .. I need to penetrate Emil but I did not appreciate .... Can you help me

**Comment by aljokarmohamed** [comment]
Please help me ...

**Pull request merged**